com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Settings/SerializedHDRenderPipelineGlobalSettings.cs

```diff
@@ -0,0 +1,197 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor.Rendering;
+using UnityEngine.Rendering.HighDefinition;
+using UnityEditorInternal; //ReorderableList
+using UnityEngine; //ScriptableObject
+using UnityEngine.Rendering; //CoreUtils.Destroy
+using System; //Type
+
+namespace UnityEditor.Rendering.HighDefinition
+{
+    class SerializedHDRenderPipelineGlobalSettings
+    {
+        public SerializedObject serializedObject;
+
+        public SerializedProperty renderPipelineResources;
+        public SerializedProperty renderPipelineRayTracingResources;
+
+        public SerializedFrameSettings defaultFrameSettings;
+        public SerializedFrameSettings defaultBakedOrCustomReflectionFrameSettings;
+        public SerializedFrameSettings defaultRealtimeReflectionFrameSettings;
+
+        public SerializedProperty volumeProfileDefault;
+        public SerializedProperty volumeProfileLookDev;
+
+        public SerializedProperty lightLayerName0;
+        public SerializedProperty lightLayerName1;
+        public SerializedProperty lightLayerName2;
+        public SerializedProperty lightLayerName3;
+        public SerializedProperty lightLayerName4;
+        public SerializedProperty lightLayerName5;
+        public SerializedProperty lightLayerName6;
+        public SerializedProperty lightLayerName7;
+
+        public SerializedProperty decalLayerName0;
+        public SerializedProperty decalLayerName1;
+        public SerializedProperty decalLayerName2;
+        public SerializedProperty decalLayerName3;
+        public SerializedProperty decalLayerName4;
+        public SerializedProperty decalLayerName5;
+        public SerializedProperty decalLayerName6;
+        public SerializedProperty decalLayerName7;
+
+        public SerializedProperty shaderVariantLogLevel;
+        public SerializedProperty lensAttenuation;
+        public SerializedProperty diffusionProfileSettingsList;
+
+        //RenderPipelineResources not always exist and thus cannot be serialized normally.
+        bool? m_hasEditorResourceHasMultipleDifferentValues;
```

Comment:
You should name `m_Has...` the h is uppercase. 

[Code Style Reference](https://ono.unity3d.com/unity-extra/unity-meta/raw/@/ReferenceSource/CSharp/Assets/CSharpReference.cs)